pom.xml

```diff
@@ -487,6 +493,12 @@
         <groupId>commons-httpclient</groupId>
         <artifactId>commons-httpclient</artifactId>
         <version>${httpclient.classic.version}</version>
+        <exclusions>
```

Comment:
I don't think we have to exclude this if we always override with a direct dependency.
There are exceptions, because sometimes SBT's different dependency resolutions cause it to pick different versions and we have to force it with an exclusion. If you don't know we need this, I think it can be removed.

Reply:
OK. I will revert it.